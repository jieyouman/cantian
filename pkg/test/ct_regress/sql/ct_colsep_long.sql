--colsep
drop table if exists wg_colsep;
create table wg_colsep (f1 int, f2 int, f3 int);
insert into wg_colsep values(1,1,1);
insert into wg_colsep values(2,2,2);
insert into wg_colsep values(3,3,3);
set colsep "
set colsep '
set colsep "asdf
set colsep 'asd
select * from wg_colsep;
set colsep ','
select * from wg_colsep;
set colsep ","
select * from wg_colsep;
set colsep "|||"
select * from wg_colsep;

--long
set long 9000
