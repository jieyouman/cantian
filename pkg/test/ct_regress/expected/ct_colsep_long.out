

SQL> --colsep
SQL> drop table if exists wg_colsep;

Succeed.

SQL> create table wg_colsep (f1 int, f2 int, f3 int);

Succeed.

SQL> insert into wg_colsep values(1,1,1);

1 rows affected.

SQL> insert into wg_colsep values(2,2,2);

1 rows affected.

SQL> insert into wg_colsep values(3,3,3);

1 rows affected.

SQL> set colsep "

String """ missing terminating quote (")
Usage: SET colsep {'text'|"text"|text}.

SQL> set colsep '

String "'" missing terminating quote (')
Usage: SET colsep {'text'|"text"|text}.

SQL> set colsep "asdf

String ""asdf" missing terminating quote (")
Usage: SET colsep {'text'|"text"|text}.

SQL> set colsep 'asd

String "'asd" missing terminating quote (')
Usage: SET colsep {'text'|"text"|text}.

SQL> select * from wg_colsep;

F1           F2           F3          
------------ ------------ ------------
1            1            1           
2            2            2           
3            3            3           

3 rows fetched.

SQL> set colsep ','


SQL> select * from wg_colsep;

F1          ,F2          ,F3          
------------,------------,------------
1           ,1           ,1           
2           ,2           ,2           
3           ,3           ,3           

3 rows fetched.

SQL> set colsep ","


SQL> select * from wg_colsep;

F1          ,F2          ,F3          
------------,------------,------------
1           ,1           ,1           
2           ,2           ,2           
3           ,3           ,3           

3 rows fetched.

SQL> set colsep "|||"


SQL> select * from wg_colsep;

F1          |||F2          |||F3          
------------|||------------|||------------
1           |||1           |||1           
2           |||2           |||2           
3           |||3           |||3           

3 rows fetched.

SQL> 
SQL> --long
SQL> set long 9000


SQL> 


